src/driver/odoc_units_of.mli

```diff
@@ -1,9 +1,14 @@
 open Odoc_unit
 
+type indices_style =
+| Voodoo
+| Normal
+| Custom
```

Comment:
The Custom mode generates index pages in every places it needs. Could it be called Automatic ?